README.md

```diff
@@ -75,8 +79,32 @@ NOTE: We recommend that we start with the defaults, updating just the URL and gi
 - `kubectl -n opsmx-isd logs isd-spinnaker-halyard-0 -c create-halyard-local`
 
 ## Check the installation
-13. Access ISD using the URL specified in the values.yaml in step 5 in a browser such as Chrome.
-14. Login to the ISD instance with user/password as admin and opsmxadmin123, if using the defaults for build-in LDAP.
+14. Access ISD using the URL specified in the values.yaml in step 5 and step 6 in a browser such as Chrome.
```

Comment:
any reason to keep these new steps/sections here ? It would have good to separate it out in keep in the supporting doc... there we can also capture some screenshots if needed